tools/storyteller/StorytellerUiTool.java

```diff
@@ -94,18 +92,25 @@ public static void main(String[] args) {
 
     checkInputArgs(args);
 
+    StorytellerComponent storytellerComponent = DaggerStorytellerComponent.builder().build();
+    Storyteller storyteller = storytellerComponent.getStoryteller();
+    StorytellerConfig storytellerConfig = storytellerComponent.getStorytellerConfig();
```

Comment:
You already get `StorytellerConfig` in the constructor, so why get it also through a component?